docs/overview.md

```diff
@@ -112,6 +112,7 @@ A brief overview of the Authorization API calls are as follows, see the live [Sw
 |Path|Method|Summary|
 |----|------|-------|
 |/authorization/access/{userId}/{action}/{resource*}|GET|Authorize user action against a resource|
+|/authorization/batchaccess/{userId}|POST|Authorize user actions against resources|
```

Comment:
wouldn't be better to say `authorize a batch of user actions`?

Reply:
updated route to just access and docs too :-)